lib/ansible/modules/system/service_facts.py

```diff
@@ -89,7 +91,8 @@ def gather_services(self):
 
         # sysvinit
         if service_path is not None and chkconfig_path is None:
-            rc, stdout, stderr = self.module.run_command("%s --status-all 2>&1 | grep -E \"\\[ (\\+|\\-) \\]\"" % service_path, use_unsafe_shell=True)
+            rc, stdout, stderr = self.module.run_command(
+                "%s --status-all 2>&1 | grep -E \"\\[ (\\+|\\-) \\]\"" % service_path, use_unsafe_shell=True, environ_update=LANG_C)
```

Comment:
instead of updating each command just set `run_command_environ_update`

Reply:
Thx, I changed this and also added LC_ALL to the override. When I understand `man 7 locale` correctly, only setting `LC_ALL` should suffice since it has the highest priority.

Reply:
man page says that .. but not all programs obey that, though i think all the ones related to services do.